ios/Classes/BetterPlayerPlugin.m

```diff
@@ -276,6 +270,13 @@ - (void) stopOtherUpdateListener: (BetterPlayer*) player{
 
 }
 
+// Refactor this function when there is more than 1 offline video
+- (BOOL)isPipPlanLimitedVideo: (BetterPlayer*) player{
```

Comment:
We should add another flag to `datasource` like `shouldClearPreviousNotificationInfo` for making clear condition to check this case